src/@types/styled-components.d.ts

```diff
@@ -38,6 +39,7 @@ declare module 'styled-components' {
         body: string;
         card: string;
         footer: string;
+        headerTransparent: string;
```

Comment:
We have a nested hierarchy in this theme file. In the future I think it makes sense to resolve camel case into the hierarchy. Instead of `headerTransparent` I would like to see `header:{ transparent: string }` for example.